src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/BlobListenerFactory.cs

```diff
@@ -53,92 +51,21 @@ internal class BlobListenerFactory : IListenerFactory
             ITriggeredFunctionExecutor executor,
             SingletonManager singletonManager)
         {
-            if (hostIdProvider == null)
-            {
-                throw new ArgumentNullException("hostIdProvider");
-            }
-
-            if (queueConfiguration == null)
-            {
-                throw new ArgumentNullException("queueConfiguration");
-            }
-
-            if (blobsConfiguration == null)
-            {
-                throw new ArgumentNullException("blobsConfiguration");
-            }
-
-            if (exceptionHandler == null)
-            {
-                throw new ArgumentNullException("exceptionHandler");
-            }
-
-            if (blobWrittenWatcherSetter == null)
-            {
-                throw new ArgumentNullException("blobWrittenWatcherSetter");
-            }
-
-            if (messageEnqueuedWatcherSetter == null)
-            {
-                throw new ArgumentNullException("messageEnqueuedWatcherSetter");
-            }
-
-            if (sharedContextProvider == null)
-            {
-                throw new ArgumentNullException("sharedContextProvider");
-            }
-
-            if (trace == null)
-            {
-                throw new ArgumentNullException("trace");
-            }
-
-            if (hostAccount == null)
-            {
-                throw new ArgumentNullException("hostAccount");
-            }
-
-            if (dataAccount == null)
-            {
-                throw new ArgumentNullException("dataAccount");
-            }
-
-            if (container == null)
-            {
-                throw new ArgumentNullException("container");
-            }
-
-            if (input == null)
-            {
-                throw new ArgumentNullException("input");
-            }
-
-            if (executor == null)
-            {
-                throw new ArgumentNullException("executor");
-            }
-
-            if (singletonManager == null)
-            {
-                throw new ArgumentNullException("singletonManager");
-            }
-
-            _hostIdProvider = hostIdProvider;
-            _queueConfiguration = queueConfiguration;
-            _blobsConfiguration = blobsConfiguration;
-            _exceptionHandler = exceptionHandler;
-            _blobWrittenWatcherSetter = blobWrittenWatcherSetter;
-            _messageEnqueuedWatcherSetter = messageEnqueuedWatcherSetter;
-            _sharedContextProvider = sharedContextProvider;
-            _trace = trace;
+            _hostIdProvider = hostIdProvider ?? throw new ArgumentNullException("hostIdProvider");
```

Comment:
>hostIdProvider [](start = 81, length = 14)

nit: as long as we're using fancy new features here ... we could use nameof(hostIdProvider) rather than the string literal.

Reply:
Yeah, I noticed that as well right when I sent this PR. These were all done with VS auto-correct so I didn't notice that it didn't auto-correct to `nameof()`. But I agree with you --  I'll update all of these with my next set of changes.